src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -11,6 +11,8 @@ namespace System.Net.Quic
 {
     public sealed class QuicConnection : IDisposable
     {
+        public static bool IsSupported => QuicImplementationProviders.Default.IsSupported;
```

Comment:
How will that work on unsupported platforms? Wouldn't QuicImplementationProviders.Default throw PNSE? Just curious